ziti/enroll/enroll.go

```diff
@@ -116,7 +116,7 @@ func Enroll(enFlags EnrollmentFlags) (*config.Config, error) {
 	if strings.TrimSpace(enFlags.KeyFile) != "" {
 		stat, err := os.Stat(enFlags.KeyFile)
 
```

Comment:
Are you going to add a check for the windows specific error?

Reply:
Was not planning on it - no. That's 'the other option' but there's no nice way to do it (imo) so checking stat for nil is just as good for me...

Reply:
ok, makes sense.